_authentication.md

```diff
@@ -198,9 +199,8 @@ Parameter   | Required | Description
 ----------- | -------- | -----------------------------------------
 description | yes      | A human-readable description of this PAT.
 
-<aside class="notice">
-  <strong>Important Notice</strong>: This request must be authenticated via OAuth, and a valid OTP token via the `OTP-Token` header.
-</aside>
+<aside class="notice">Requires the <code>OTP-Method-Id</code> header to be sent with the id of a verified authentication method that belongs to the user.</aside>
+<aside class="notice">Requires the <code>OTP-Token</code> header to be sent with a valid TOTP token, belonging to the authentication method specified in <code>OTP-Method-Id</code>.</aside>
```

Comment:
Here too I'm not sure the "belonging" wording works. Would "generated with" work better?